schema/2.0.0/corpora_schema.md

```diff
@@ -312,7 +312,7 @@ Curators MUST annotate the following columns in the `obs` dataframe:
               <td>A term from the set of <a href="http://www.ontobee.org/search?ontology=MMUSDV&keywords=month-old&submit=Search+terms"> month-old stages</a><br>(e.g. <a href="http://purl.obolibrary.org/obo/MmusDv_0000062">MmusDv:0000062)</a></td>
             </tr>
           </tbody></table>
-          <br> Otherwise, for all other organisms this MUST be the most accurate child of <a href="http://www.ebi.ac.uk/efo/EFO_0000399"<code>EFO:0000399</code></a> for <i>development_stage</i>.
+          <br> Otherwise, for all other organisms this MUST be the most accurate child of <a href="http://purl.obolibrary.org/obo/UBERON:0000105"<code>UBERON:0000105</code></a> for <i>life cycle stage</i>, excluding <a href="http://purl.obolibrary.org/obo/UBERON_0000071"<code>UBERON:0000071</code></a> for <i>death stage</i>, <a href="http://purl.obolibrary.org/obo/UBERON_0035944"<code>UBERON:0035944</code></a> for <i>life-death temporal boundary</i>, and <a href="http://purl.obolibrary.org/obo/UBERON_0035945"<code>UBERON:0035945</code></a> for <i>start of life cycle</i>.
```

Comment:
http://purl.obolibrary.org/obo/UBERON:0000105 --> http://purl.obolibrary.org/obo/UBERON_0000105
`life-death temporal boundary` & `start of life cycle` are not children of `life cycle stage` so can be removed from the exclusion list (they are children of `life cycle` [UBERON:0000104])

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/117/commits/dda43abfe2f2f63b2764aa928c5ee348e4c634dc